packages/app/features/home/screen.tsx

```diff
@@ -27,16 +30,32 @@ export function HomeScreen() {
           community of beatmakers!
         </P>
         <View className="android:w-60 ios:w-60 mx-auto">
-          <SignUpButton />
-          <LogInButton />
-          <TryOutButton />
+          {!!user && <GoToDashboardButton />}
+          {!user && <SignUpButton />}
+          {!user && <LogInButton />}
+          {!user && <TryOutButton />}
         </View>
       </View>
       <Footer />
     </View>
   );
 }
 
+function GoToDashboardButton() {
```

Comment:
I've just noticed that all those buttons reuse the same style. This component was written a long time ago and since then we introduced `Button` component in the design system. Could you add a new variant to `Button` component with this style and use it here for all those buttons?